supervisor/shared/board_busses.h

```diff
@@ -29,7 +29,8 @@
 
 #include "py/obj.h"
 
-mp_obj_t board_i2c(void);
```

Comment:
The declaration of `board_i2c()` got deleted here, though `board_spi()` and `board_uart()` still exist. Was that deliberate?

Reply:
Ya, I only switched one. Will push a larger refactor shortly.